nix/wrapper.nix

```diff
@@ -48,13 +50,76 @@ let
         fi
       '';
     };
+  sanitizedWrapperName =
```

Comment:
Need a comment here explaining why we need this sanitization in the first place. Is it because one or more of the shells can't work with special characters like `,`, or?